pyartcd/pyartcd/pipelines/build_microshift_bootc.py

```diff
@@ -219,18 +224,14 @@ async def _rebuild_needed():
                                      f" {url}, but could not find it. Use --force to rebuild plashet.")
 
                 microshift_nvrs = await get_microshift_builds(self.group, self.assembly, env=self._elliott_env_vars)
-                expected_microshift_nvr = next((n for n in microshift_nvrs if isolate_el_version_in_release(n) == 9), None)
-                if not expected_microshift_nvr:
-                    message = f"Could not find el9 microshift nvr for assembly {self.assembly}. Please investigate"
-                    self._logger.info(message)
-                    raise ValueError(message)
+                expected_microshift_nvr = next(n for n in microshift_nvrs if isolate_el_version_in_release(n) == 9)
                 if actual_nvr != expected_microshift_nvr:
                     self._logger.info(f"Found nvr {actual_nvr} in plashet.yml is different from expected {expected_microshift_nvr}. Plashet build is needed.")
                     return True
                 self._logger.info(f"Plashet has the expected microshift nvr {expected_microshift_nvr}")
                 return False
 
-        rebuild_needed = self.force_plashet_sync or await _rebuild_needed()
+        rebuild_needed = await _rebuild_needed() or self.force_plashet_sync
```

Comment:
Checking `self.force_plashet_sync` before calling `_rebuild_needed()` looks reasonable for performance. Could you explain why you swap them?

Reply:
Yeah I had suggested that to [Sid](https://redhat-internal.slack.com/archives/GDBRP5YJH/p1741720533936809?thread_ts=1741718399.405609&cid=GDBRP5YJH). Looks like he got the change in, and my PR still had the old code. Will fix it, thanks